### ISSUE TITLE
Visual bug: Missing text on boss health bar

### ISSUE THREAD

**Issue by askebv** [Heishe/idleraiders-second-run#511]
![image](https://cloud.githubusercontent.com/assets/4981659/23705051/10f58a16-0408-11e7-916c-c3b9238563f3.png)

I'm not sure exactly how it happened (sorry). I had a boss selected, went to a different browser tab and then came back and it was like this. And now it's like this all the time. So far it differs whether it's the second or third bar, but haven't seen the first bar have problems. Refreshing the page fixes it.

**Comment by Meissner-Effect** [comment]
Yes, this has happened to me as well on the same boss set when coming back from a different tab. When it happened to me, there was neither a health bar nor the text for the health of Ox2 (just blank) even though it had full health.

Edit: Now the third boss tab seems to be permanently empty across all raids (here is an example with Ox3):

![capture](https://cloud.githubusercontent.com/assets/26352345/23833419/1068b52c-073d-11e7-974c-245bbf9771b8.PNG)
